fe/fe-core/src/main/java/com/starrocks/common/ErrorCode.java

```diff
@@ -254,6 +254,8 @@ public enum ErrorCode {
                     ErrorCode.ERR_ACCESS_DENIED_HINT_MSG_FORMAT),
     ERR_ACCESS_DENIED_FOR_EXTERNAL_ACCESS_CONTROLLER(5204, new byte[] {'4', '2', '0', '0', '0'},
             "Access denied; you need (at least one of) the %s privilege(s) on %s%s for this operation."),
+    ERR_SECURE_TRANSPORT_REQUIRED(5205, new byte[] {'H', 'Y', '0', '0', '0'},
```

Comment:
is this error code just defined by starrocks, or is there a corresponding error code in mysql, so sr mimics mysql's error code as much as possible.

Reply:
@kevincai, error code updated based on @gengjun-git comments here: https://github.com/StarRocks/starrocks/pull/56176#discussion_r1970800912

Reply:
ok